src/devices/SocketCan/SocketCan.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <!--Disabling default items so samples source won't get build by the main library-->
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <LangVersion>Latest</LangVersion>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="*.cs" />
+    <None Include="README.md" />
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
```

Comment:
Please reference the property version instead here, so that you don't have a wildcard reference.

Reply:
@joperezr side topic, but related... I'm assuming you're talking about the following?

```xml
<PackageReference Include="System.Device.Gpio" Version="$(SystemDeviceGpioPackageVersion)" />
```

I get build errors when using this local on other projects.  What should be updated to get the projects to build correctly?

Reply:
> I'm assuming you're talking about the following?

Yup, that's what I'm talking about.

> I get build errors when using this local on other projects. What should be updated to get the projects to build correctly?

When you say other projects do you mean projects outside of the repo? If so, that is expected, since this is a variable that is defined in our repo which is imported by all projects in the repo, so it won't work for projects that are outside of it. Here is where we define it:

https://github.com/dotnet/iot/blob/26e1b07557ce650d3110769262a27b50f38650be/eng/Versions.props#L5

If you see, we update this guy daily with an auto-generated PR everytime we have a new build of our packages.

Reply:
> When you say other projects do you mean projects outside of the repo?

That's what it was.  Thanks as usual!